Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -913,6 +913,7 @@
     <string name="submission_intro_illustration_description">"Ein positiver Testbefund wird verschlüsselt ins System übermittelt, das nun andere Nutzerinnen und Nutzer warnt."</string>
     <!-- YTXT: submission introduction bullet points -->
     <string-array name="submission_intro_bullet_points">
+        <item>"Wenn sich auf Ihrem Testdokument ein QR-Code befindet, können Sie diesen in der App scannen und den Test damit registrieren. Sobald das Ergebnis vorliegt, können Sie es in der App einsehen."</item>
```

Comment:
Testdokument? Oder eher: Testformular? Das ist doch dieses Formular, auf dem man auch ankreuzen muss, das man der elektronischen Übermittlung des Ergebnisses via App zustimmt, oder? Der ist nicht auf dem Befund.

Reply:
Will be adddressed separately